src/plugins/intel_cpu/src/nodes/kernels/aarch64/jit_uni_eltwise_generic.cpp

```diff
@@ -345,6 +379,24 @@ void jit_uni_eltwise_generic<isa>::load_scalar(const SReg& data,
             ldr(data, Xbyak_aarch64::ptr(ptr, ptr_offset));
             break;
         }
+        case ov::element::i8: {
+            ldr(Xbyak_aarch64::BReg(data.getIdx()), Xbyak_aarch64::ptr(ptr, ptr_offset));
+
+            // scalar is loaded, operates with vector
+            TReg vec(data.getIdx());
```

Comment:
nit: I was wondering if we can do shuffling in the code block below (which does precision conversion) for consistancy with load_vector routine.